voidcrew/modules/shuttle/helm/_helm.dm

```diff
@@ -83,11 +85,8 @@
 
 	return data
 
-/*
-
 /obj/machinery/computer/helm/LateInitialize()
 	. = ..()
- //voidcrew todo: ship functionality
 	reload_ship()
```

Comment:
I'm not entirely sure what to do with ``reload_ship``

Shuttles use ``connect_to_shuttle()`` on everything post load, but I don't think it links up anything that is built on the shuttle?

We can probably move this to computer helm's Initialize and use ``connect_to_shuttle(port = SSshuttle.get_containing_shuttle(src))`` instead of a new proc, I don't think we need to reload which ship we're on every ui_act.

I understand if you don't want to do this here because it already is a large PR, this is more of a note to myself for a future one.
It also will be a problem for RD consoles atm lol

Reply:
Yeah, the issue is if a helm is built on a ship it wont have connect_to_shuttle called on it.
`reload_ship` just acts as a "if we aren't linked, try to link"

Reply:
Changed the way it worked to prevent a get container call if not needed, and some basic error checking